cli/requirements.txt

```diff
@@ -15,13 +15,15 @@ aws-cdk.aws-ssm~=1.164
 aws-cdk.core~=1.164
 aws_cdk.aws-cloudwatch~=1.164
 aws_cdk.aws-lambda~=1.164
-boto3>=1.16.14
+boto3>=1.16.14,<1.33.2
```

Comment:
May you please add a comment linking to the known issue behind this pinning?
I also suggest to track a task in the backlog to remove this pinning for the next release.